### PR TITLE
Expose array transforms and array contains queries.

### PR DESCRIPTION
**Pull request by mikelehen**
Also remove test code that was combining multiple array contains queries since those were disallowed in 04c9c3a.

### CODE REVIEW

File: packages/firestore/CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # Unreleased
+- [feature] Added `firebase.firestore.FieldValue.arrayUnion()` and
+  `firebase.firestore.FieldValue.arrayRemove()` to atomically add and remove
+  elements from an array field in a document.
+- [feature] Added `'array-contains'` query operator for use with `.where()` to
```
Comment (schmidt-sebastian) [inline packages/firestore/CHANGELOG.md]: Is the double-quoting intentional for better formatting?

File: packages/firestore/test/integration/api/array_transforms.test.ts [hunk 2]
```diff
@@ -21,20 +21,16 @@ import firebase from '../util/firebase_export';
 import { apiDescribe, withTestDoc, withTestDb } from '../util/helpers';
 import { EventsAccumulator } from '../util/events_accumulator';
 
-// TODO(array-features): Remove "as any"
 // tslint:disable-next-line:no-any variable-name Type alias can be capitalized.
```
Comment (schmidt-sebastian) [inline packages/firestore/test/integration/api/array_transforms.test.ts]: You can remove "no-any" from here.

Comment (mikelehen) [reply packages/firestore/test/integration/api/array_transforms.test.ts]: oops, thanks!

File: packages/firestore/CHANGELOG.md (hunk 1 above)
Comment (mikelehen) [reply packages/firestore/CHANGELOG.md]: yep

**Comment by mikelehen** [comment]
Josh, can you approve for the d.ts change?

**Pull request merged**